src/sentry/api/endpoints/project_metrics_extraction_rules.py

```diff
@@ -71,7 +71,9 @@ def get(self, request: Request, project: Project) -> Response:
             on_results=lambda x: serialize(
                 x, user=request.user, serializer=MetricsExtractionRuleSerializer()
             ),
-            default_per_page=25,
+            default_per_page=1000,
+            max_per_page=1000,
+            max_limit=1000,  # overrides default max_limit of 100 when creating paginator object
```

Comment:
This is ok for now, but I am pretty sure we will need to limit the number of extraction rules per project, as they have a big impact on the size of project configs sent to relay.